lib/node_modules/@stdlib/math/base/special/cexp/test/test.js

```diff
@@ -115,128 +108,154 @@ tape( 'the function computes exp(z) for complex z', function test( t ) {
 	expim = generalComplex.expim;
 
 	for ( i = 0; i < re.length; i++ ) {
-		q = cexp( re[ i ], im[ i ] );
-		if ( q[ 0 ] === expre[ i ] ) {
-			t.strictEqual( q[ 0 ], expre[ i ], 'returns expected real component' );
+		q = cexp( new Complex128( re[ i ], im[ i ] ) );
+		if ( real( q ) === expre[ i ] ) {
+			t.strictEqual( real( q ), expre[ i ], 'returns expected real component' );
 		} else {
-			delta = abs( q[ 0 ] - expre[ i ] );
+			delta = abs( real( q ) - expre[ i ] );
 			tol = EPS * abs( expre[ i ] );
-			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. real: '+q[0]+'. expected: '+expre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. real: '+real( q )+'. expected: '+expre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
 		}
-		if ( q[ 1 ] === expim[ i ] ) {
-			t.strictEqual( q[ 1 ], expim[ i ], 'returns expected imaginary component' );
+		if ( imag( q ) === expim[ i ] ) {
+			t.strictEqual( imag( q ), expim[ i ], 'returns expected imaginary component' );
 		} else {
-			delta = abs( q[ 1 ] - expim[ i ] );
+			delta = abs( imag( q ) - expim[ i ] );
 			tol = EPS * abs( expim[ i ] );
-			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. imag: '+q[1]+'. expected: '+expim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. imag: '+imag( q )+'. expected: '+expim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
 		}
 	}
 	t.end();
 });
 
 tape( 'if imaginary component is `+Infinity`, the function returns `[NaN, NaN]`', function test( t ) {
-	var v = cexp( 0.0, PINF );
-	t.strictEqual( isnan( v[ 0 ] ), true, 'returns NaN' );
-	t.strictEqual( isnan( v[ 1 ] ), true, 'returns NaN' );
+	var v;
+
+	v = cexp( new Complex128( 0.0, PINF ) );
+	t.strictEqual( isnan( real( v ) ), true, 'returns NaN' );
+	t.strictEqual( isnan( imag( v ) ), true, 'returns NaN' );
 	t.end();
 });
 
 tape( 'if imaginary component is `-Infinity`, the function returns `[NaN, NaN]`', function test( t ) {
-	var v = cexp( 0.0, NINF );
-	t.strictEqual( isnan( v[ 0 ] ), true, 'returns NaN' );
-	t.strictEqual( isnan( v[ 1 ] ), true, 'returns NaN' );
+	var v;
+
+	v = cexp( new Complex128( 0.0, NINF ) );
+	t.strictEqual( isnan( real( v ) ), true, 'returns NaN' );
+	t.strictEqual( isnan( imag( v ) ), true, 'returns NaN' );
 	t.end();
 });
 
 tape( 'if real component is `-Infinity`, the function returns `[0.0, 0.0]`', function test( t ) {
-	var v = cexp( NINF, 1.0 );
-	t.strictEqual( v[ 0 ], 0.0, 'returns 0.0' );
-	t.strictEqual( v[ 1 ], 0.0, 'returns 0.0' );
+	var v;
+
+	v = cexp( new Complex128( NINF, 1.0 ) );
+	t.strictEqual( real( v ), 0.0, 'returns 0.0' );
+	t.strictEqual( imag( v ), 0.0, 'returns 0.0' );
 	t.end();
 });
 
 tape( 'if real component is `+Infinity` and imaginary component is `0.0`, the function returns `[+Infinity, 0.0]`', function test( t ) {
```

Comment:
This and the following descriptions should be updated, as the function no longer returns an array.